tests/unit/cli/admin/test_ceremony.py

```diff
@@ -332,16 +329,13 @@ def test_ceremony_start_default_values(
             "strongPass",
             "tests/files/timestamp1.key",
             "strongPass",
-            "tests/files/JoeCocker.key",
-            "strongPass",
             "tests/files/bins1.key",
             "strongPass",
             "y",
             "y",
             "y",
             "y",
             "y",
-            "y",
```

Comment:
Removing one `y` as an input can be done as the last six `y` inputs are used to answer on the question wheter a role configuration are correct and given we don't use the `BIN` role anymore we can remove one `y`.